### PR TITLE
Proper colors when asm.filter=false

### PR DESCRIPTION
**Pull request by kazarmy**
Before:
asm_filter_before
After:
asm_filter_after

### CODE REVIEW

**Comment by radare** [comment]
can you write a test for this regression? thanks!

File: libr/core/disasm.c [hunk 1]
```diff
@@ -755,9 +755,7 @@ static void ds_build_op_str(RDisasmState *ds) {
 			free (ds->opstr);
 			ds->opstr = strdup (ds->str);
 		} else {
-			if (!ds->opstr) {
-				ds->opstr = strdup (asm_str? asm_str: "");
-			}
+			ds->opstr = strdup (asm_str? asm_str: "");
```
Comment (radare) [inline libr/core/disasm.c]: but this is leaking

Comment (radare) [inline libr/core/disasm.c]: fixed in master

**Comment by kazarmy** [comment]
Test in radare/radare2-regressions#793.

**Comment by radare** [comment]
Thanks!

**Pull request merged**